### PR TITLE
feat(docs): Add `bucket_id` to start message properties

### PR DESCRIPTION
**Pull request by andrewswait**
Noticed this wasn't mentioned and is important if you want the engine to look at your specific (possibly private) service group.

Screenshot of locally run docs with this change:
image

### CODE REVIEW

File: pages/apis/ai-engine/chat.mdx [hunk 1]
```diff
@@ -166,6 +166,11 @@ There are 4 types of messages that can be sent to the backend:
       description:
         "The session ID, obtained from the Create Session API, uniquely identifies a session",
     },
+    {
+      name: "bucket_id",
```
Comment (qati) [inline pages/apis/ai-engine/chat.mdx:170]: can we add this to the session creation bit pls? as thats where this can be actually created

Comment (andrewswait) [reply pages/apis/ai-engine/chat.mdx:170]: Not seeing `bucket_id` being passed when the session is created (unless I'm missing something). Here's the request payload that the DeltaV FE is sending even though a service group has been selected to `POST
	https://agentverse.ai/v1beta1/engine/chat/sessions`
 
image

Am I missing something?

Comment (devjsc) [reply pages/apis/ai-engine/chat.mdx:170]: @qati to follow up please 👍

**Comment by andrewswait** [comment]
Just noticed this is still hanging around in my open PRs so am resurrecting 😄  happy to just close if not relevant any more @qati

**Pull request closed without merging**